src/repos/WeeklyDebriefRepo.ts

```diff
@@ -0,0 +1,55 @@
+import { DocumentType } from '@typegoose/typegoose';
+import { User, UserModel } from '../entities/User';
+import WeeklyDebrief from '../entities/WeeklyDebrief';
+import { ArticleModel } from '../entities/Article';
+
+const getExpirationDate = (creationDate: Date): Date => {
+  creationDate.setDate(creationDate.getDate() + 7);
```

Comment:
what happens when `creationDate.getDate()` is 29? does `setDate` automatically roll those extra days over to the next month?

Reply:
A different note, on this same line of code I didn't think about, didn't we decide there was a uniform day in which all weekly debrief objects were created? In which case shouldn't they all expire at the same time too? I could be forgetting the logic we decided on, but if this is the case, it shouldn't be `getDate() + 7` but rather setting to the next upcoming "sunday" or whatever time weekly debriefs are created on.

Reply:
`setDate` does automatically go to the next month in that case. Also what specific day and time should we have the universal expiration date happen?

Reply:
So I've been looking around for a solution using the Date() class and I can't really find any way to do it with a specific date. Wouldn'rt having the `getDate() + 7`  portion run on the cronjob be sufficient? Currently new users have a null weekly debrief, so all expiration dates should be uniform

Reply:
I'm not sure how we'd handle timezones, so ignoring that for now, I was able to find [this](https://www.w3schools.com/jsref/jsref_getday.asp) to get the day of the week. If I'm understanding @tedim52 correctly and we want weekly debriefs to update on Sundays, something like this:
`const currentTime = new Date();
const daysUntilSunday = 7 - currentTime.getDay();
creationDate.setDate(currentTime.getDate() + daysUntilSunday);
creationDate.setHours(9);
creationDate.setMinutes(0);`
could work (it would set the creationDate to be 9am on the next Sunday). Unsure if that's what y'all are looking for though.

Reply:
This is resolved. The cronjob runs once a week, so `getDate() + 7` would set expirationDate to the same time for all cronjobs as desired.